WalletWasabi.Fluent/Views/Settings/PrivacySettingsTabView.axaml

```diff
@@ -8,6 +8,14 @@
              x:DataType="settings:PrivacySettingsTabViewModel"
              x:CompileBindings="True">
   <StackPanel Classes="settingsLayout">
+    <StackPanel.Styles>
+      <Style Selector="DockPanel > ToggleSwitch">
+        <Setter Property="Margin" Value="0 0 -12 0" />
+        <Setter Property="HorizontalAlignment" Value="Right" />
+        <Setter Property="OnContent" Value="{x:Null}" />
+        <Setter Property="OffContent" Value="{x:Null}" />
+      </Style>
+    </StackPanel.Styles>
```

Comment:
You can remove this part safely. The line above `Classes="settingsLayout"` will take care of this.

Reply:
Removed.